### DISCUSSION TITLE
Osu!lazer not starting without Administrator priviledges

### DISCUSSION THREAD

**Discussion by MikaL42** [ppy/osu#23152] [Q&A] [answered]
So i downloaded osu!lazer earlier today, and it was working just about the same as normal osu had, until after a restart it has completely stopped opening unless i open it with administrator priviledges, which as far as osu!lazer itself says, is not recommended, might anyone know ways to try to fix this?

**Comment by peppy** [comment]
Check windows error log for any issues, or provide error logs if you can.

We need something to work with to help you further.

**Reply by MikaL42** [reply]
Heres some error logs i cooked up, hope they'll help.
[Squirrel.log](https://github.com/ppy/osu/files/11182471/Squirrel.log)
![image](https://user-images.githubusercontent.com/99755004/230702389-c25ba7b6-94a2-44f6-a92d-0cb1c7e81c21.png)

**Reply by peppy** [reply, marked as answer]
You need to fix the permissions on your install. It seems like you installed (incorrectly) as administrator.

You can google for instructions on how to do this.

**Reply by MikaL42** [reply]
Oh i see, thanks for the help
